src/worldcereal/openeo/extract.py

```diff
@@ -49,6 +54,8 @@ def post_job_action_patch(
     title: str,
     spatial_resolution: str,
     s1_orbit_fix: bool = False,  # To rename the samples from the S1 orbit
+    write_stac_api: bool = False,
+    sensor: str = "PATCH_SENTINEL1",
```

Comment:
this value is different from the `"Sentinel1"` in `extract.py`. Is that intended?